include/envoy/json/json_object.h

```diff
@@ -136,7 +136,7 @@ class Object {
 
   /**
    * Validates JSON object against passed in schema.
-   * @param schema supplies the schema in string format. A Json::Exception will be thrown, if
+   * @param schema supplies the schema in string format. A Json::Exception will be thrown if
```

Comment:
I already commented on this, but the exception will also be thrown if the schema itself is not valid. Please update docs.